VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-participant-status-list/judge-participant-status-list.component.spec.ts

```diff
@@ -124,9 +136,38 @@ describe('JudgeParticipantStatusListComponent', () => {
 
         expect(logger.error).toHaveBeenCalled();
     });
+
+    it('should update new staff member display name with user input', () => {
+        const newName = 'new name';
+        component.onEnterStaffMemberDisplayName(newName);
+        expect(component.newStaffMemberDisplayName).toBe(newName);
+    });
+
+    it('should save new staff member display name in database', async () => {
```

Comment:
This doesn't test it saves to DB just that we call the function on videoWebService.

Reply:
I'll correct the name